File: functions/isprime.py
def isprime(event,context):
  n= event['data']
  if n == 2 or n == 3: return "Prime"
  if n < 2 or n%2 == 0: return "Not Prime"
  if n < 9: return "Prime"
  if n%3 == 0: return "Not Prime"
  r = int(n**0.5)
  f = 5
  while f <= r:
    print '\t',f
    if n%f == 0: return "Not Prime"
    if n%(f+2) == 0: return "Not Prime"
    f +=6
  return "Prime"

File: functions/isprime.js
module.exports = {
  handler: (event, context) => {
    console.log(event);
    num=event.data;
    if (num == 1) return "Not Prime";
  num += 2;

  var upper = Math.sqrt(num);
  var sieve = new Array(num)
    .join(',').split(',') // get values for map to work
    .map(function(){ return true });
  
  for (var i = 2; i <= num; i++) {
    if (sieve[i]) {
      for (var j = i * i; j < num; j += i) {
        sieve[j] = false;
      };
    };
  };
if (sieve[num-2]) {
  return "Prime";
  };
  else {
  return "Not Prime";
};
  },
};
